code/modules/mob/living/living_defines.dm

```diff
@@ -52,4 +52,9 @@
 	var/list/datum/action/actions = list()
 	var/step_count = 0
 
-	var/list/surgeries = list()	//a list of surgery datums. generally empty, they're added when the player wants them.
\ No newline at end of file
+	var/list/surgeries = list()	//a list of surgery datums. generally empty, they're added when the player wants them.
+	
+	//smell vars
+	var/datum/smell/current_smell = null //last sensed smell
+	var/constipated = 0 //set by viruses, used to tell if the nose is clogged or not, clogged noses cant smell

```

Comment:
This is the var name we would add when we implement toilets actually being usable.